modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -63,16 +63,17 @@ $ az identity show -g <cluster_resource_group> \// <1>
 <1> Specifies the name of the cluster resource group created by the installation program.
 <2> Specifies the name of the cluster service principal created by the installation program.
 The identity is in the format of `12345678-1234-1234-1234-1234567890`.
-. Create a role assignment that grants the cluster service principal `Contributor` privileges to the disk encryption set by running the following command:
+. Create a role assignment that grants the cluster service principal privileges of a custom role to the disk encryption set by running the following command:
```

Comment:
How about 

Create a role assignment that grants the cluster service principal necessary privileges to the disk encryption set by running the following command:


@jinyunma WDYT ?

Reply:
looks better, since user can decide to grants system role or customer role to disk encryption set.

Reply:
Addressed